CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.2.5
+  - Fix failling tests introduce by the `ssl_key_passphrase` changes.
```

Comment:
@guyboertje Do not restraint yourself for grammar changes, I like learning that way.